packages/xdl/src/start/ExpoUpdatesManifestHandler.ts

```diff
@@ -114,9 +114,11 @@ export async function getManifestResponseAsync({
 
   const hostUri = await UrlUtils.constructHostUriAsync(projectRoot, hostname);
 
+  const isManagedProject = getDefaultTarget(projectRoot) === 'managed';
```

Comment:
this is actually a bit problematic because eas-cli and expo-cli have different ideas of what this means. we want to move away from classifying managed/bare projects in expo-cli, so i think we shouldn't add this.

for eas-cli, **we need to know if a project has ios/android directories committed to git**. if not, then we read data and do validations differently, and we run prebuild on eas build.

because eas update is an eas service, it should follow that same logic imo.

Reply:
Ah, and here I was thinking this was solved!

Thanks for point this out.